tests/frontend/unit/DpClaim.spec.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * (c) 2010-present DEMOS E-Partizipation GmbH.
+ *
+ * This file is part of the package demosplan,
+ * for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+import { createLocalVue } from '@vue/test-utils'
+import DpClaim from '@DpJs/components/statement/DpClaim'
+import shallowMountWithGlobalMocks from '@DpJs/VueConfigLocal'
+
+describe('DpClaim', () => {
+  it('should be an object', () => {
+    expect(typeof DpClaim).toBe('object')
+  })
+
+  it('should be named DpClaim', () => {
+    expect(DpClaim.name).toBe('DpClaim')
+  })
+
+  it('should return the correct icons and text for fragments-claim-states', async () => {
+    const localVue = createLocalVue()
+
+    const wrapper = shallowMountWithGlobalMocks(DpClaim, {
+      propsData: {
+        assignedOrganisation: 'Orga des Assingnee',
+        assignedName: 'Aktueller Benutzer',
+        assignedId: '',
+        currentUserId: '1',
+        currentUserName: 'Aktueller Benutzer',
+        lastClaimedUserId: '',
+        entityType: 'fragment'
+      },
+      localVue
+    })
+
+    // Should show open lock if assignedId is not set - lastClaimed doesn't matter.
+    expect(wrapper.vm.status).toEqual({ icon: 'fa-unlock', text: 'statement.fragment.assignment.unassigned' })
```

Comment:
I think each `expect` statement should be moved to a separate `it` block, but I'm not sure it has to happen in this PR, since we just want to restore the tests asap.